layersvt/CMakeLists.txt

```diff
@@ -111,29 +39,36 @@ else()
     add_compile_options(-Wpointer-arith)
 endif()
 
-find_package(Python3 REQUIRED)
-
-set(VULKANTOOLS_SCRIPTS_DIR "${VULKAN_TOOLS_SOURCE_DIR}/scripts")
-set(VULKAN_REGISTRY "${VULKAN_HEADERS_INSTALL_DIR}/${CMAKE_INSTALL_DATADIR}/vulkan/registry")
-
-# Define macro used for building vk.xml generated files
-function(run_vulkantools_vk_xml_generate dependency output)
-    add_custom_command(OUTPUT ${output}
-        COMMAND Python3::Interpreter -B ${VULKANTOOLS_SCRIPTS_DIR}/vt_genvk.py -registry ${VULKAN_REGISTRY}/vk.xml -scripts ${VULKAN_REGISTRY} ${output}
-        DEPENDS ${VULKAN_REGISTRY}/vk.xml ${VULKAN_REGISTRY}/generator.py ${VULKANTOOLS_SCRIPTS_DIR}/${dependency} ${VULKANTOOLS_SCRIPTS_DIR}/vt_genvk.py ${VULKAN_REGISTRY}/reg.py
-    )
-endfunction()
+if(BUILD_APIDUMP)
```

Comment:
Moved all the API dump code inside this block which makes it MUCH clearer what is happening.